### ISSUE TITLE
Use GeForce RTX 4090 with Intel NUC

### ISSUE THREAD

**Issue by yoshikiohshima** [ajbouh/substrate#158]
some additional adapters and cables are needed, and some more tinkering seems to be needed to use GeForce 4090 with Intel NUC.

**Comment by yoshikiohshima** [comment]
It was suggested to get one https://c-payne.com/products/slimsas-pcie-device-adapter-2-8i-to-x16, one https://c-payne.com/products/pcie-slimsas-host-adapter-x16-to-2-8i and two https://c-payne.com/products/slimsas-sff-8654-8i-cable-straight-pcie-gen4-85ohm?pr_prod_strat=jac&pr_rec_id=638929443&pr_rec_pid=7977418686731&pr_ref_pid=6587753169101&pr_seq=uniform

**Comment by mgood** [comment]
To clarify: it sounds like this is to add it as an external GPU? If you want to install it as the main GPU in the case, it does work, though the power connector is a tight fit. I ended up leaving off the bottom panel on the case. A 90 degree power adapter would solve that, though one of the major manufactures did have a recall recently due to a fire risk, so we'd want to be careful about which ones to use.

**Comment by yoshikiohshima** [comment]
Thanks @mgood I thought you also had a solution, as I saw conversation on DIscord.

**Comment by ajbouh** [comment]
@mgood are you certain you're talking about the same GPU as @yoshikiohshima?

**Comment by yoshikiohshima** [comment]
either way, if you can help me overcome my inability to search things on Discord and give me the link to the relevant conversation, that would be great.

**Comment by mgood** [comment]
Yeah, I guess you already have a 4060 or something in the case? So IIUC those devices would let you use both of them as external GPUs through the PCI adapters.
In the meantime if you want to replace the 4060 with the 4090 it should work ok. However, the Nvidia adapter needed to provide sufficient power isn't very flexible and sticks out of the bottom of the case. I didn't want to risk breaking it, so I turned the case up onto the front panel which seems pretty sturdy, and left the bottom panel off like this:
![IMG_2859](https://github.com/ajbouh/substrate/assets/29016/4586a6e7-1f76-4f9c-ae26-627ec9bd238e)

**Comment by ajbouh** [comment]
@mgood I believe you are assuming @yoshikiohshima has a Nvidia RTX 4090 Founders Edition. (He does not.)

**Comment by mgood** [comment]
Ah, ok, yeah that's the detail that was missing here. I guess the others are both too big, and maybe require too much power for the NUC?

**Comment by ajbouh** [comment]
Primarily size, yes. Here's a list of sizes in case you're curious: https://www.reddit.com/r/nvidia/comments/xzphl7/4090s_comparison_chart/
